style.css

```diff
@@ -0,0 +1,50 @@
+body{
+    padding:0;
+    margin:0;
+    font-family:"Arial",sans-serif;
+}
+
+
+.container{
+    width:100%;
+    max-width:1024px;
+    margin-left:auto;
+    margin-right:auto;
+    padding:10px;
+    background-color: rgba(233, 249, 45, 0.22);
+}
+
+a{
+    color: rgba(23, 174, 0, 0.78);
+    text-decoration:none;
+}
+
+a:hover{
+    box-shadow:0 0px 5px 0 black;
+}
+
+.photo{
+    float:right;
+    width:150px;
```

Comment:
After the colon in the rules put one space.
See more in this guide: http://codeguide.academy/html-css.html#html-language